helpick-service/src/test/java/com/infoshare/service/VolunteerServiceTest.java

```diff
@@ -0,0 +1,29 @@
+package com.infoshare.service;
+
+import com.infoshare.database.DB;
+import com.infoshare.database.FileDb;
+import com.infoshare.domain.TypeOfHelp;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+
+
+class VolunteerServiceTest {
+
+    private final DB db = new FileDb();
+    private final VolunteerService volunteerService = new VolunteerService(db);
+
+    @Test
+    public void ifNewVolunteerRegisteredForValidInput() {
+        assertTrue(volunteerService.registerNewVolunteer("Janko", "Gdansk", "janko@gmail.com", "123654789", TypeOfHelp.SHOPPING, true));
```

Comment:
Nie powinniśmy zaczynać od asercji, tylko na niej kończyć 😉 
Warto pamiętać o strukturze testu (sekcje Given/When/Then) i w jednym teście sprawdzać tylko jedną metodę -> w tym wypadku tylko registerNewVolunteer, a już nie deleteVolunteer (na to osobny test w klasie DbTest). Przykład:
```
@Test
public void ifNewVolunteerRegisteredForValidInput() {
    // Given & When
    boolean isVolunteerRegistered = volunteerService.registerNewVolunteer("Janko", "Gdansk", "janko@gmail.com", "123654789", TypeOfHelp.SHOPPING, true);

    // Then
    assertThat(isVolunteerRegistered).isTrue();
}
```